docs/source/troubleshooting/troubleshooting-getting-started.adoc

```diff
@@ -138,3 +138,14 @@ Downloading https://github.com/minishift/minishift/releases/download/v1.2.0/mini
  65 B / 65 B [===================================================================================================================================================] 100.00% 0s
 Update failed: open /usr/bin/.minishift.new: permission denied
 ----
+
+[[safari-openshift-web-console-issue]]
+== Safari browser issue with OpenShift web console
+
+OpenShift web console URL i.e. `minishift console` does not work on older versions of Safari web browser e.g. version 10.1.2 (12603.3.8) and its gives following error:
+
+----
+Error unable to load details about the server
+----
+
+Re-try after updating it to the latest version or use Firefox or Chrome browser for this. You can use `minishift console --url` to get the web console URL.
```

Comment:
It would be good if we specify the version which works here.

```
Re-try after updating it to the latest version like bla bla or use Firefox. ...
```